celesta-sql/src/main/java/ru/curs/celesta/score/MaterializedView.java

```diff
@@ -304,7 +294,7 @@ void addFromTableRef(TableRef ref) throws ParseException {
 
     public String getChecksum() {
         try (ChecksumInputStream is = new ChecksumInputStream(
```

Comment:
@packpaul: Unwanted coupling here: `CelestaSerializer` was not intended to be used from GrainElement classes. 

I looked at the code and I see no easy way to get around this problem, so maybe this place will be a technical debt.

Reply:
**This is still an unwanted coupling.** Maybe a code comment should be added here, something like 
```java
//TODO: CelestaSerializer is not intended to be used from GrainElement classes.
```

Reply:
I'v got no other ideas on how to calculate the checksum, so I'll put a TODO: clause.

Reply:
👌